cmd/draftd/start.go

```diff
@@ -119,9 +124,17 @@ func (c *startCmd) run() (err error) {
 		cfg.UseTLS = true
 		cfg.TLSConfig = tlscfg
 	}
+	switch c.storageEngine {
+	case "configmaps":
+		const namespace = "default"
```

Comment:
Should the namespace be taken from the `draft.toml` file instead of defaulting to "default"?

Reply:
Seems reasonable. Likely it will be pulled from the draftd chart value overrides after draft init.